Entity/Dynamic.php

```diff
@@ -361,9 +361,9 @@ public function getDate(): ?string
     }
 
     /**
-     * @return string
+     * @return string|mixed[]|null
```

Comment:
@alexander-schranz I'm not sure what this method will return in all states of a request ... I know in a event listener for `FormSavePostEvent`, this will return `int[]`

Reply:
I think this is always `null|int[]` and attachment is never a string, as the method is not used anywhere in the core this did not yet throw an error.

Reply:
fixed